examples/node_pool/main.tf

```diff
@@ -63,7 +63,7 @@ module "gke" {
 
   additional_ip_ranges_config = [
     {
-      subnetwork           = var.subnetwork
+      subnetwork           = "projects/${var.project_id}/regions/${var.region}/subnetworks/${var.subnetwork}"
```

Comment:
@apeabody I think this was the issue, plan passes either way. but error is strange